fdbclient/NativeAPI.actor.cpp

```diff
@@ -4150,40 +4173,27 @@ Future<RangeResultFamily> getExactRange(Reference<TransactionState> trState,
 	}
 }
 
-Future<Key> resolveKey(Reference<TransactionState> trState,
-                       KeySelector const& key,
-                       Version const& version,
-                       UseTenant useTenant) {
+Future<Key> resolveKey(Reference<TransactionState> trState, KeySelector const& key, UseTenant useTenant) {
 	if (key.isFirstGreaterOrEqual())
 		return Future<Key>(key.getKey());
 
 	if (key.isFirstGreaterThan())
 		return Future<Key>(keyAfter(key.getKey()));
 
-	return getKey(trState, key, version, useTenant);
+	return getKey(trState, key, useTenant);
 }
 
 ACTOR template <class GetKeyValuesFamilyRequest, class GetKeyValuesFamilyReply, class RangeResultFamily>
 Future<RangeResultFamily> getRangeFallback(Reference<TransactionState> trState,
-                                           Version version,
                                            KeySelector begin,
                                            KeySelector end,
                                            Key mapper,
                                            GetRangeLimits limits,
                                            int matchIndex,
                                            Reverse reverse,
                                            UseTenant useTenant) {
-	if (version == latestVersion) {
```

Comment:
I couldn't find anywhere that these `latestVersion` requests were happening. If there is somewhere that they are used, though, we can add back support for them.